Projects/Example Projects/ModCreatorKit/ThumbnailCaptureScript.cpp

```diff
@@ -43,6 +43,26 @@ ThumbnailCaptureScript::~ThumbnailCaptureScript()
 {
 }
 
+//---------------------------------------------------
+// Detours
+
+PaletteUIPtr pLastPalette;
```

Comment:
Don't use managed pointer for this, use `Palettes::PaletteUI` directly. Otherwise, there might be problems with memory (as this pointer will only be cleared after the game is closed)